aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -620,12 +712,30 @@ public AwsProperties() {
   public AwsProperties(Map<String, String> properties) {
```

Comment:
In my first commit, I got CI failure because:
```
[ERROR] /home/runner/work/iceberg/iceberg/aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java:714:3: Method AwsProperties length is 154 lines (max allowed is 150). [MethodLength]
```
Currently, I get around this issue by deleting some blank lines in `AwsProperties(Map<String, String> properties)`. I wonder if there exist a better way to handle this issue, given that we may need to add more properties in the future.